src/test/java/org/broadinstitute/hellbender/tools/spark/pathseq/PathSeqScoreIntegrationTest.java

```diff
@@ -43,11 +43,11 @@ private static boolean testPathogenTaxonScores(final PSPathogenTaxonScore a, fin
             final PSPathogenTaxonScore score = new PSPathogenTaxonScore();
             final String taxonId = tok[0];
             score.setKingdomTaxonId(Math.abs(tok[4].hashCode()));
-            score.addSelfScore(new Double(tok[5]));
-            score.addScoreNormalized(new Double(tok[6]));
-            score.addTotalReads(new Integer(tok[7]));
-            score.addUnambiguousReads(new Integer(tok[8]));
-            score.setReferenceLength(new Long(tok[9]));
+            score.addSelfScore(Double.valueOf(tok[5]));
```

Comment:
this should really be `Double.parseDouble` instead but it's just a test so it doesn't really matter